proto/staking/v1beta1/querier.proto

```diff
@@ -1,14 +1,48 @@
 syntax = "proto3";
 package heimdall.staking.v1beta1;
 
-import "cosmos/base/query/v1beta1/pagination.proto";
-// this line is used by starport scaffolding # 1
+import "heimdall/types/validator.proto";
```

Comment:
We should also change `validator.signer `to `AccAddress` from `string` if that's not a problem with GRPC. 

@anilCSE what do you think?

Reply:
gRPC gateway doesn't have support for `casttype` annotation, I will think about some alternative for this. For now let's keep it as is and we'll address this in a separate PR altogether

Reply:
I have created issue #461 for this. Let's take it up in separate PR.